### DISCUSSION TITLE
Is this layout possible?

### DISCUSSION THREAD

**Discussion by evbrew** [nhn/tui.chart#660] [Ideas]
I don't think there is currently, but is there any way to achieve these 3 visual components?

(see picture below for bullet # references)
1. Control the padding between axis markers and data area?

2. Show graph Data top aligned above all elements.

3. Style the "empty" area between edge of graph and Data element.
Could probably cheat and do this with a dummy stacked data element but ...

![GraphExample](https://user-images.githubusercontent.com/29763631/115908782-74799a80-a41f-11eb-858f-c34e443c0cbc.png)

**Comment by jung-han** [comment]
This option is not currently available.
If you can tell me what kind of chart the example chart below is, I'll take a look.
